pulp_container/app/models.py

```diff
@@ -186,7 +189,39 @@ class Meta:
         unique_together = (("name", "tagged_manifest"),)
 
 
-class ContainerNamespace(BaseModel, AutoAddObjPermsMixin, AutoDeleteObjPermsMixin):
+class CreateGroupAndAddPermissionsMixin(AutoAddObjPermsMixin):
```

Comment:
Nice!
Should we have the same for group removal? To ensure that no perms are inherited from the removed objects with the same field value?
It's critical for namespaces if we use its name and not pk.

Reply:
+1 it would be good to remove groups and perms whenever namespace/distribution also gets deleted.

Reply:
We should. I only considered the removal of the permissions which already occurs. However, the groups themselves need to be removed also. I'll add another mixin for that.